web/viewer.css

```diff
@@ -1084,7 +1093,6 @@ html[dir='rtl'] .verticalToolbarSeparator {
 }
 
 .toolbarLabel {
-  min-width: 16px;
```

Comment:
Is it actually necessary to remove this line?
If I understand correctly, you changed this in order to avoid adding a bunch of empty space at the right side of the `findbar` when `findMsg` isn't set. If that's correct, wouldn't adding the following rule achieve that in a better/simpler way?
```css
#findMsg:empty {
  display: none;
}
```